.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -120,11 +161,25 @@ for (config in configs) {
     for (_version in versions) {
         def version = _version
 
-        def combinedName = "${label}-${version}"
-        builders[combinedName] = {
-            node(label) {
-                stage(combinedName) {
-                    build(version, label)
+        if (label.contains("docker")) {
+            def imageName = config["imageName"]
+            def combinedName = "${imageName}-${version}"
+            builders[combinedName] = {
+                node(label) {
+                    stage(combinedName) {
+                        docker.image(imageName).inside("-u root") {
```

Comment:
(Turns out the manylinux1 images don't have virtualenv so if we switched away from root we'd need to add the new user to the container, install virtualenv, and then init and source a venv in the bash script. It was decided this was more complexity than just letting it operate as root)